GameServerLib/Logic/Logger.cs

```diff
@@ -89,14 +119,19 @@ public void Log(string lines, string type = "LOG")
                     type.ToUpper(),
                     lines
                 );
-                lock (_locker)
+
+                _stringBuilder.AppendLine(text);
+
+                Task.Factory.StartNew(() =>
                 {
-                    File.AppendAllText(
-                        _logFileName,
-                        text + Environment.NewLine
-                    );
-                    Console.WriteLine(text);
-                }
+                    lock (_locker)
+                    {
+                        File.AppendAllText(
```

Comment:
How about we move the file writing to the refresh loop as well? Just make sure to flush it on fatal errors before exitting